tests/Session/test_commands.py

```diff
@@ -113,6 +114,83 @@ def testAddFood(self, mocked_input):
         # reset standout
         sys.stdout = sys.__stdout__
 
+    @patch('src.Session.commands.input', create=True)
+    def testRemoveFood(self, mocked_input):
+        ## no profile ##
+        # grab print output
+        capturedPrintOutput = io.StringIO()
+        sys.stdout = capturedPrintOutput
+
+        self.commands.remove_food()
+
+        expectedOutput = \
+            NO_LOADED_PROFILE_MESSAGE + " " + \
+            SUGGESTED_ACTIONS_MESSAGE.format("'{}', '{}'".format(CREATE_PROFILE, LOAD)) + NEW_LINE
+        self.assertEqual(expectedOutput, capturedPrintOutput.getvalue())
+       
+        # reset standout
+        sys.stdout = sys.__stdout__
+
+        ## Negative path: food storage not open ##
+        mocked_input.side_effect = [profile_name]
+        self.commands.create_profile()
+
+        # grab print output
+        capturedPrintOutput = io.StringIO()
+        sys.stdout = capturedPrintOutput
+        self.commands.remove_food()
+
+        expectedOutput = FOOD_OPEN_ERROR_MESSAGE + NEW_LINE + \
+            SUGGESTED_ACTIONS_MESSAGE.format(listInQuotes([OPEN_FRIDGE, OPEN_FREEZER, CREATE_FREEZER, CREATE_FRIDGE])) + NEW_LINE
+        self.assertEqual(expectedOutput, capturedPrintOutput.getvalue()) 
+
+        # reset standout
+        sys.stdout = sys.__stdout__
+
+        ## Success path: food storage exists, is open, 2 foods added, 1 food removed ##
+        mocked_input.side_effect = [freezer_name]
+        self.commands.create_foodStorage(FREEZER)
+        mocked_input.side_effect = [name1, date1, use_by_date1, YES, name2, date2, EMPTY_STRING, NO]
+        self.commands.add_food()
+
+        # grab print output
+        capturedPrintOutput = io.StringIO()
+        sys.stdout = capturedPrintOutput
+
+        mocked_input.side_effect = [name1, date1, use_by_date1, NO]
+        self.commands.remove_food()
+
+        actualOutput = capturedPrintOutput.getvalue().strip().split(NEW_LINE)[-1]
+        self.assertEqual(REMOVE_FOOD_SUCCESS_MESSAGE_COMPLETE, actualOutput)
+
+        testInventory = self.commands.profile.getFoodStorage(FREEZER, freezer_name).list()
+
+        self.assertNotIn(name1, testInventory)
```

Comment:
I think this is supposed to be `food1`? I thought `name1` was a string and rather than a Food

Reply:
changed it!